support-files/mariadb@.service.in

```diff
@@ -73,19 +73,19 @@ PermissionsStartOnly=true
 # Do not panic if galera_recovery script is not available. (MDEV-10538)
 ExecStartPre=/bin/sh -c "systemctl unset-environment _WSREP_START_POSITION%I"
 
-ExecStartPre=/bin/sh -c "[ ! -e /usr/bin/galera_recovery ] && VAR= || \
- VAR=`/usr/bin/galera_recovery --defaults-file=@INSTALL_SYSCONF2DIR@/my%I.cnf`; [ $? -eq 0 ] \
+ExecStartPre=/bin/sh -c "[ ! -e @prefix@/bin/galera_recovery ] && VAR= || \
+ VAR=`@prefix@/bin/galera_recovery --defaults-file=@sysconf2dir@/my%I.cnf`; [ $? -eq 0 ] \
```

Comment:
I'm thinking '@prefix@/bin' should be '@bindir@'. galera_recovery is part of SYSTEMD_SCRIPTS which in scripts/CMakeLists.txt gets added to BIN_SCRIPTS before 'INSTALL_SCRIPT(...DESTINATION ${INSTALL_BINDIR}' a small distance later in the same file.

Otherwise great improvement and apologies for my previous laziness that you are now correcting :+1: